src/olympia/devhub/templates/devhub/addons/submit/sidebar.html

```diff
@@ -4,13 +4,23 @@
 {# List of steps: (text, class). A class of 'all' means the step is relevant
    to listed or unlisted addons, a class of 'listed' means it's only relevant
    to listed addons, not unlisted ones. #}
-{% set NAV = ((_('Getting Started'), 'all'),
+{% if waffle.flag('no-prelim-review') %}
+    {% set NAV = ((_('Getting Started'), 'all'),
+              (_('Upload your add-on'), 'all'),
+              (_('Describe your add-on'), 'all'),
+              (_('Add images'), 'listed'),
+              (_('Select a license'), 'listed'),
+              (_('n/a'), 'hidden'),
```

Comment:
No you're right, it's ugly but it's probably the best solution for the time being.